frontend/src/app/main/data/workspace/selection.cljs

```diff
@@ -84,8 +84,10 @@
 
 ;; --- Toggle shape's selection status (selected or deselected)
 
-(defn select-shape
-  ([id] (select-shape id false))
+(defn select-shape-
```

Comment:
Why the `-` suffix on the function name?

Reply:
ups! it's something i did when i was refactoring and i forgot to undo. I'll revert it